File: core/src/main/java/vn/loitp/flickr/model/photosetgetphotos/WrapperPhotosetGetPhotos.java

package vn.loitp.flickr.model.photosetgetphotos;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class WrapperPhotosetGetPhotos {

    @SerializedName("photoset")
    @Expose
    private Photoset photoset;
    @SerializedName("stat")
    @Expose
    private String stat;

    public Photoset getPhotoset() {
        return photoset;
    }

    public void setPhotoset(Photoset photoset) {
        this.photoset = photoset;
    }

    public String getStat() {
        return stat;
    }

    public void setStat(String stat) {
        this.stat = stat;
    }

}

File: core/src/main/java/com/loitp/restApi/flickr/model/photoSetGetPhotos/WrapperPhotosetGetPhotos.kt
package com.loitp.restApi.flickr.model.photoSetGetPhotos

import androidx.annotation.Keep
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import com.loitp.core.base.BaseModel

/**
 * Created by Loitp on 04,August,2022
 * Galaxy One company,
 * Vietnam
 * +840766040293
 * freuss47@gmail.com
 */
@Keep
class WrapperPhotosetGetPhotos : BaseModel() {
    @SerializedName("photoset")
    @Expose
    var photoset: Photoset? = null

    @SerializedName("stat")
    @Expose
    var stat: String? = null
}
